meson.build

```diff
@@ -2,7 +2,23 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-project('opentitan', 'c', 'cpp', version: '0.1',)
+project(
+  'opentitan', 'c', 'cpp',
+  version: '0.1',
+  meson_version: '>=0.51.0',
```

Comment:
This now also fixes https://github.com/lowRISC/opentitan/issues/1101. Please add a comment here and in python-requirements.txt that both places need to be updated.

Reply:
Done